src/extensions/observersTitleBuilderMap.test.ts

```diff
@@ -18,6 +18,8 @@ describe('observersComponentBuilderMap', () => {
     ['azureDevOps', { alias: 'awesome alias' }, 'Azure DevOps: awesome alias'],
     ['opsgenie', { host: 'host', identifier: 'id' }, 'Opsgenie: host/id'],
     ['opsgenie', { alias: 'awesome alias' }, 'Opsgenie: awesome alias'],
+    ['bitbucket', { workspace: 'workspace', repo: 'repo', branch: 'branch' }, 'Bitbucket: repo/branch'],
```

Comment:
Should this expected string be 'Bitbucket: workspace/repo/branch' as people could have more than one workspace?

Reply:
sure! I can also add it. I thought it would be redundant since you have enough info with the repo itself.

Reply:
Ah just seen you added it, thanks!